modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent-based installer
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments. You can generate an ISO that contains all the technology required to run both a user friendly guided deployment and a fully automated one.
+The Agent installation comprises of a bootable ISO that contains the Assisted discovery agent and the Assisted service; both are required to perform the first cluster installation.
+
+The `openshift-install agent create` subcommand generates an ephemeral ISO that is generated based on the inputs that you provide. You can choose to provide inputs through the following manifests:
+
+* install-config.yaml
+* agent-config.yaml
+
+or
+
+* cluster-deployment.yaml
+* agent-cluster-install.yaml
+* pull-secret.yaml
+* infraenv.yaml
+* cluster-image-set.yaml
+* nmstateconfig.yaml
+
+[id="agent-based-installer-workflow"]
+== Agent-based installer workflow
+One of the nodes runs the assisted service at the start of the boot process and eventually becomes the bootstrap host. This node is **node 0**.
+The assisted service ensures that all the hosts meet the requirements and triggers a {product-title} cluster deployement. All the nodes have the Red Hat Enterprise Linux CoreOS (RHCOS) image written to the disk. The non bootstrap nodes reboot and initiate a cluster deployment.
+Once the nodes are rebooted, **node 0** reboots and joins the cluster. The bootstrapping is complete and the cluster is deployed.
+
+image::agent-based-installer-workflow.png[Agent-based installer workflow]
+
+You can install a disconnected {product-title} cluster through the `openshift-install agent create image` subcommand for:
+
+* **A single-node {product-title} cluster (SNO)**: A node that is both a master and worker.
+* **A three-node {product-title} cluster** : A compact cluster that has three master nodes that are also worker nodes.
+* **Highly available {product-title} cluster (HA)**: Three master nodes with any number of worker nodes.
+
+The following platforms are supported:
+
+* none
+* bare-metal
+* vSphere
```

Comment:
baremetal
vsphere

Reply:
we have references of vSphere in docs.openshift repo and in official VMware , term is vSphere.